requirements/requirements_trtllm.txt

```diff
@@ -0,0 +1,5 @@
+tiktoken
+jiwer
```

Comment:
why is this required?

Reply:
tiktoken for whisper official tokenizer. jiwer for the repo's wer metric computation.